susemanager-utils/susemanager-sls/src/modules/mgrnet.py

```diff
@@ -0,0 +1,137 @@
+"""
+Module for gathering DNS FQDNs
+"""
+
+import logging
+import re
+import time
+from threading import Thread
+from queue import Queue
+
+import salt.utils.network
+import salt.utils.path
+import salt.utils.platform
```

Comment:
I don't know the details enough, but why is this imported when not used?

Reply:
I'll check, thanks for pointing :+1: . Maybe I just forgot to remove something what was used before one of the modifications.

Reply:
Thanks @aaannz . There was a code using this import before in `__virtual__`, but I've removed it as redundant.